rsts/deployment/sandbox.rst

```diff
@@ -145,6 +145,10 @@ Flyte configuration on your remote cluster.
 
 #. Open the Kubernetes dashboard http://<K8S_DASHBOARD_URL>.
 
+#. Port-forward to connect minio to flyteadmin using the following command: ::
```

Comment:
Port-forward to connect minio from your localhost using the following command(Run in background): ::